modules/images/webp-uploads/load.php

```diff
@@ -683,3 +683,91 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 	return rest_ensure_response( $data );
 }
 add_filter( 'rest_prepare_attachment', 'webp_uploads_update_rest_attachment', 10, 3 );
+
+/**
+ *
+ * Updates webp image when original image is edited
```

Comment:
This isn't specific to WebP, it will generate image files for any formats specified in the filter from `webp_uploads_get_upload_image_mime_transforms()`.
```suggestion
 * Creates additional image formats when original image is edited.
```